examples/retry-transaction-statement/retry_transaction_statement.md

```diff
@@ -1,10 +1,9 @@
 # Retry transaction statement
 
 Transactional errors are often transient: retrying will fix them.
-This works by creating a  RetryManager object `r`, before executing the transaction.
-If the block fails with error `e`, it calls `r.shouldRetry(e)`.
-If that returns `true`, then it executes the block again.
-`retry` has an optional type parameter giving class of `RetryManager` to create, and optional arguments to new `DefaultRetryManager` tries `n` times.
+This works by creating a `RetryManager` object `r`, before executing the transaction.
+If the block fails with error `e`, it calls `r.shouldRetry(e)`. If that returns `true`, then it executes the block again.
+`retry` has an optional type parameter giving class of `RetryManager` to create, and optional arguments to new. `DefaultRetryManager` tries 3 times.
```

Comment:
```suggestion
`retry` has an optional type parameter giving class of `RetryManager` to create, and optional arguments to new. `DefaultRetryManager` tries 3 times.
```
The idea is not clear and is grammatically incorrect. Needs to rewrite.

Reply:
Updated in 48f771040254c32ba752611f95d753cbb6dfa925